lib/mongo/database.rb

```diff
@@ -171,18 +171,19 @@ def collections
     #
     # @return [ Mongo::Operation::Result ] The result of the command execution.
     def command(operation, opts = {})
-      execution_opts = opts.delete(:execution_options) || {}
+      opts_copy = opts.dup
```

Comment:
This can simply be `opts = opts.dup` since generally the original argument won't be used past this point.